Example/PrebidDemoJava/src/main/AndroidManifest.xml

```diff
@@ -11,14 +11,14 @@
     <!-- Optional permissions. Used for MRAID 2.0 storePicture ads -->
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
     <application
-        android:networkSecurityConfig="@xml/network_security_config"
```

Comment:
What is the reason of removing `android:networkSecurityConfig` ?

Reply:
Mistake, will add it back.

Reply:
Switched to simple config: android:usesCleartextTraffic="true"